src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -1,219 +1,232 @@
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
```

Comment:
It seems like you've accidentally imported an icon from it's individual module, instead of importing it with the other icons from `@mui/icons-material` (on line 8-13). Try to figure out how this happens, so that you can avoid it in the future.